cli/tests/integrations/helpers/common.py

```diff
@@ -46,7 +46,9 @@ def exec_command(cmd, env=None, stdin=None, timeout=None):
         # process and finish communication.
         # https://docs.python.org/3.5/library/subprocess.html#subprocess.Popen.communicate
         process.kill()
-        process.communicate()
+        stdout, stderr = process.communicate()
+        print('STDOUT: {}'.format(_truncate(stdout.decode('utf-8'))))
```

Comment:
We don't typically use `print()` directly (we use `emitter` instead).  Also, I'm not sure if we want to unconditionally print this stuff. Should we only print it when we have log_level=debug?

Reply:
This function is only run from our tests suites, the stdout / stderr are hided by default and only displayed in a context of a test failure, for debugging purpose. A few lines below we are also printing directly (in case the process exited with non-zero).

Reply:
i agree with @klueska, we should not print out the debugging output in tests, would be better to put asserts here, and if assert triggers, then print out the output

Reply:
Are we talking about the same thing here? I agree that we shouldn't print directly and this is not printing directly, by default pytest captures stdout / stderr for each test and only displays it in case of a test failure (cf. https://docs.pytest.org/en/latest/capture.html#default-stdout-stderr-stdin-capturing-behaviour).

We are already printing stdout/stderr for all commands (cf. a few lines below), this is very helpful to troubleshoot a failure in jenkins by looking at the console output for example.